node/src/components/linear_chain_sync/operations.rs

```diff
@@ -280,7 +309,7 @@ async fn get_trusted_key_block_info(
 async fn fetch_and_store_block_by_height(
```

Comment:
It would be nice to deduplicate this and `fetch_and_store_block_header_by_height` at some point. (Not necessarily in this PR.)

Reply:
I've thought about this.  We'd want:

- A `HasFinalitySignatures` trait
- A `Storable` trait and a generic `effect_builder::put::<T :: Storable>` function.